spec/tbd_tests_spec.rb

```diff
@@ -2468,54 +2488,289 @@
       n_edges_as_corners           += 1 if edge[:type] == :corner
       n_edges_as_concave_corners   += 1 if edge[:type] == :cornerconcave
       n_edges_as_convex_corners    += 1 if edge[:type] == :cornerconvex
       n_edges_as_transitions       += 1 if edge[:type] == :transition
     end
 
+    # Lo Scrigno holds 8x wall/roof edges:
+    #   - 4x along gallery roof/skylight (all convex)
+    #   - 4x along the elevator roof (3x convex + 1x concave)
+    #
+    # The gallery wall/roof edges are not modelled here "as built", but rather
+    # closer to details of another Renzo Piano extension: the Modern Wing of the
+    # Art Institute of Chicago. Both galleries are similar in that daylighting
+    # is zenithal, covering all (or nearly all) of the roof surface. In the
+    # case of Chicago, the roof is entirely glazed (as reflected in the model):
+    #
+    # www.archdaily.com/24652/the-modern-wing-renzo-piano/
+    # 5010473228ba0d42220015f8-the-modern-wing-renzo-piano-image?next_project=no
+    #
+    # This is allowed in OpenStudio, yet SketchUp (via the plugin) would
+    # complain.
+    #
+    # No judgement here on the suitability of the design for either Chicago or
+    # Turin. The model nonetheless remains an interesting (~extreme) test case
+    # for TBD. Here, the transition from "wall-to-roof" and "roof-to-skylight"
+    # are one and the same. So is the edge a :skylight edge? or a :parapet (or
+    # :roof) edge? They're both. In such cases, the final selection in TBD is
+    # based on the greatest PSI factor. In ASHRAE 90.1 2022, only "vertical
+    # fenestration" edge PSI factors are explicitely stated/published. For this
+    # reason, the 8x TBD-built-in ASHRAE PSI sets have 0 W/K per meter assigned
+    # for any non-regulated edge, e.g.:
+    #
+    #   - skylight perimeters
+    #   - non-fenestrated door perimeters
+    #   - corners
+    #
+    # There are (possibly) 2x admissible interpretations of how to treat
+    # non-regulated heat losss (edges as linear thermal bridges) in 90.1:
+    #   1. assign 0 W/K•m for both proposed design and budget building models
+    #   2. assign more realistic PSi factors, equally to both proposed/budget
+    #
+    # In both cases, the treatment of non-regulated heat loss remains "neutral"
+    # between both proposed design and budget building models. Option #2 remains
+    # closer to reality (more heat loss in winter, likely more heat gain in
+    # summer), which is preferable for HVAC autosizing. Yet 90.1 (2022) ECB
+    # doesn't seem to afford this type of flexibility, contrary to the "neutral"
+    # treatment of (non-regulated) miscellaneous (process) loads. So for now,
+    # TBD's built-in ASHRAE 90.1 2022 (A10) PSI factor sets recflect option #1.
+    #
+    # Users who prefer option #2 can always write up a custom ASHRAE 90.1 (A10)
+    # PSI factor set on file (tbd.json), initially based on the built-in 90.1
+    # sets yet while resetting non-zero PSI factors.
```

Comment:
Reiterating the issue raised [here](https://github.com/rd2/tbd/pull/109/commits/4386168c7a208a1af34a537157ba0f31a84d10e4#r1335010855). Will adapt if needed (e.g. based on future 90.1 2022 User Guide or clarification from 90.1 TC).